src/segment-loader.js

```diff
@@ -258,6 +258,24 @@ export default class SegmentLoader extends videojs.EventTarget {
     this.checkBufferTimeout_ = window.setTimeout(this.monitorBuffer_.bind(this),
                                                  CHECK_BUFFER_DELAY);
   }
+
+  /**
+   * Return the amount of a segment specified by the mediaIndex overlaps
+   * the current buffered content.
+   * @param playlist {object} the playlist object to fetch segments from
+   * @param mediaIndex {number} the index of the segment in the playlist
```

Comment:
same as the previous comment https://github.com/videojs/videojs-contrib-hls/pull/629/files#r58073036